subsys/net/lib/nrf_cloud/src/nrf_cloud_fota_common.c

```diff
@@ -251,20 +251,40 @@ int nrf_cloud_fota_fmfu_apply(void)
 	err = nrf_modem_lib_bootloader_init();
 	if (err != 0) {
 		LOG_ERR("nrf_modem_lib_bootloader_init() failed: %d", err);
-		(void)nrf_modem_lib_init();
-		return err;
+
+		err = nrf_modem_lib_init();
+		if (err != 0) {
+			LOG_ERR("nrf_modem_lib_init() failed: %d", err);
+			return err;
+		}
+
+		return -ECANCELED;
 	}
 
 	err = fmfu_fdev_load(fmfu_buf, sizeof(fmfu_buf), fmfu_dev.dev, fmfu_dev.offset);
 	if (err != 0) {
 		LOG_ERR("Failed to apply full modem update, error: %d", err);
-		(void)nrf_modem_lib_init();
-		return err;
+		LOG_ERR("Putting modem back in normal mode");
```

Comment:
```suggestion
		LOG_INF("Putting modem back in normal mode");
```
i know this is in response to an error, but this is more of an informative log about what the device is doing.

Reply:
Ok. Do you prefer LOG_INF here over DBG? I tend to favour dbg in submodules. Easier to enable logging from different part of the system.

Reply:
normally i would prefer debug, but since this is only printed on an error, inf is fine.